### PR TITLE
added and placed SVGs in service card avatars

### PR DESCRIPTION
**Pull request by leseanbiz**
added images folder and files. directed .json to .svg files.

### CODE REVIEW

File: client/app/components/ServicesTemplate/ServicesCardContent/styles.scss [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 .title {
   padding-bottom: 0.25rem;
+  // This padding-top doesn't work. 
+  padding-top: .70rem !important; 
 }
```
Comment (Globolobo) [inline client/app/components/ServicesTemplate/ServicesCardContent/styles.scss]: please avoid committing !important, and/or code that doesn't appear to be working

File: client/app/components/ServicesTemplate/data.json [hunk 2]
```diff
@@ -6,48 +6,51 @@
...
+          "url":"route",
+          "img": "wireless flagging"
         },
         {
           "id": "3",
           "title": "Contact Data Enrichment",
           "subtitle": "Ensure you are reaching the people you mean to with accurate contacts. ",
-          "url":"route"
+          "url":"route",
+          "img": "contact data enrichment"
```
Comment (Globolobo) [inline client/app/components/ServicesTemplate/data.json]: avoiding using spaces when naming files, use snake_case convention

File: client/app/components/ServicesTemplate/data.json [hunk 3]
```diff
@@ -6,48 +6,51 @@
...
+          "url": "CR",
+          "img": "CR"
         },
         {
           "id": "4",
           "title": "Inter-Continental",
           "content": "CNAM, LIDB, Verification, and Status hosting services of the telecom industry",
-          "url": "telified",
-          "img": "telecom"
+          "url": "IC",
```
Comment (Globolobo) [inline client/app/components/ServicesTemplate/data.json]: lowercase only for naming files, while mac isn't case sensitive, Linux is, and it's pita to debug later

File: client/app/components/ServicesTemplate/index.js [hunk 4]
```diff
@@ -13,13 +13,15 @@ const ServicesTemplate = () => {
...
+      {...{ title,  content, url}}
     />
   );
 
-  const filledCardServices = json.data.serviceCardContent.map(({ id, title, subtitle, url }) =>
+  const filledCardServices = json.data.serviceCardContent.map(({ id, title, subtitle, url, img }) =>
     <ServicesCardContent
       key={id}
+      img={<img role="presentation" src={require(`./images/${img}.svg`)} style={{ fill: '#000066'}} />}
       {...{ title, subtitle, url }}
```
Comment (Globolobo) [inline client/app/components/ServicesTemplate/index.js]: props should only ever be variable values, if they're static like in this case, they should be moved down to the child component (role, and style)

Comment (Globolobo) [reply client/app/components/ServicesTemplate/index.js]: You should also avoid passing down full elements down to children components, and only fill missing values.

File: client/app/components/ServicesTemplate/index.js [hunk 5]
```diff
@@ -13,13 +13,15 @@ const ServicesTemplate = () => {
     <Card
       key={id}
       cardSize="is-6"
-      {...{ title,  content, url, img }}
+      img={<img role="presentation" src={require(`./images/${img}.svg`)} style={{ fill: '#000066'}} />}
+      {...{ title,  content, url}}
```
Comment (Globolobo) [inline client/app/components/ServicesTemplate/index.js]: Same as below. Generally, if you're copy pasting code, it means you can do some form of abstraction

Comment (leseanbiz) [reply client/app/components/ServicesTemplate/index.js]: i will put this in a different story and prioritize after demo.

File: client/app/components/ServicesTemplate/data.json (hunk 2 above)
Comment (stern-shawn) [reply client/app/components/ServicesTemplate/data.json]: I'm also confused why the routes are just `route` when I already went to the trouble of making the `telecom-hosting`, `wireless-flagging`, etc routes during our first pass of the site

File: client/app/components/ServicesTemplate/index.js (hunk 4 above)
Comment (stern-shawn) [reply client/app/components/ServicesTemplate/index.js]: FYI @Globolobo this is a side effect of you originally making the shared Card work that way, just sayin' :heart:

File: client/app/components/ServicesTemplate/data.json [hunk 6]
```diff
@@ -1,56 +1,53 @@
 {
-  "data":
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/data.json:2]: @leseanbiz Unnecessary nesting of fields under `data` key

File: client/app/components/ServicesTemplate/index.js [hunk 7]
```diff
@@ -3,27 +3,24 @@ import React from 'react';
...
 import ServicesCardContent from './ServicesCardContent';
-import json from './data.json';
+import data from './data.json';
 
 // Styles
 import styles from './styles.scss';
 
+// TODO - update Card to not take <img> directly, pass src string and maybe sizing/styles as props
 const ServicesTemplate = () => {
-  const filledCardsTop = json.data.topCardContent.map(({ title, content, url, img, id }) =>
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js:12]: @leseanbiz Notice how this simplifies how the JSON is used on the JavaScript side?

**Review by stern-shawn — APPROVED** [review]
After these changes, I think we're good to go. Everything looks good, code-wise I think all we could improve is updating the Card component, but that'll have effects on other parts of the app that also use Card, so @Globolobo should take that on when he can in a different branch. :+1:

**Pull request merged**